diff --git a/src/libqhull/user.h b/src/libqhull/user.h
index f1fca9a..443da82 100644
--- a/src/libqhull/user.h
+++ b/src/libqhull/user.h
@@ -144,7 +144,7 @@ Code flags --
    REALfloat =   1      all numbers are 'float' type
              =   0      all numbers are 'double' type
 */
-#define REALfloat 0
+#define REALfloat 1
 
 #if (REALfloat == 1)
 #define realT float
diff --git a/src/libqhull_r/user_r.h b/src/libqhull_r/user_r.h
index 9e0c73b..ee98532 100644
--- a/src/libqhull_r/user_r.h
+++ b/src/libqhull_r/user_r.h
@@ -143,7 +143,7 @@ Code flags --
    REALfloat =   1      all numbers are 'float' type
              =   0      all numbers are 'double' type
 */
-#define REALfloat 0
+#define REALfloat 1
 
 #if (REALfloat == 1)
 #define realT float
